laa_court_data_api_app/routers/laa_references.py

```diff
@@ -48,39 +48,39 @@ async def patch_maat_unlink(defendant_id: str, request: ExternalPatchRequest):
 
 @router.post("/v2/laa_references", status_code=202, responses=responses)
 async def post_maat_link(request: ExternalPostRequest):
-    logger.info(f"Calling_Maat_Post_{request.defendant_id}")
+    logger.info("Calling_Maat_Post", defendant_id=request.defendant_id)
     client = CourtDataAdaptorClient()
 
-    cda_response = await client.post(f"/api/internal/v2/laa_references/",
+    cda_response = await client.post("/api/internal/v2/laa_references/",
                                      body=InternalPostRequest(laa_reference=InternalPost(**request.dict())))
 
     return formulated_response(cda_response, request.defendant_id, "Linking")
 
 
 def formulated_response(cda_response, defendant_id, request_type):
     if cda_response is None:
-        logging.error("Laa_References_Endpoint_Did_Not_Return")
+        logger.error("Laa_References_Endpoint_Did_Not_Return")
```

Comment:
I'm guessing the defendant_id wouldn't help with debugging and thats why it's not included in the error log?

Reply:
Every little helps! added info